spec/lib/ability_spec.rb

```diff
@@ -66,7 +66,7 @@
       let(:facility) { Facility.cross_facility }
 
       it { is_expected.to be_allowed_to(:manage_accounts, facility) }
-      it { is_expected.to be_allowed_to(:manage, Account) }
+      it { is_expected.to be_allowed_to([:read, :edit], Account) }
```

Comment:
@jhanggi, whenever you have a sec--this spec is still failing and I can't figure out why. What am I missing?